apps/src/music/player/MusicPlayer.ts

```diff
@@ -1,3 +1,10 @@
+import {Effects} from './interfaces/Effects';
+import {FunctionContext} from './interfaces/FunctionContext';
+import {PatternEvent, PatternEventValue} from './interfaces/PatternEvent';
+import {PlaybackEvent} from './interfaces/PlaybackEvent';
+import {SkipContext} from './interfaces/SkipContext';
+import {SoundEvent} from './interfaces/SoundEvent';
+import {TrackMetadata} from './interfaces/TrackMetadata';
```

Comment:
No idea what the standard is, but I wonder if it would be helpful to have a naming decoration for interviews, e.g. a prefix of `I`?

Reply:
Also curious how things will scale with even more interfaces.. do we want lots of small files, or some consolidation?

Reply:
Hmm good point, i've seen `filename.interface.ts` used before - maybe that would be better?

Also agreed on scaling. I'm not sure if there's a good standard way to do this. One pattern I've seen is a single `types.ts` files that stores all of these interfaces, but the risk is that growing very large and hard to read. I've previously had smaller files like, but I can go either way.

Reply:
Oh, that first suggestion was actually to name things like `IEffects`, `IFunctionContext` but no idea if that's helpful or not. :)

Reply:
Oh yep! I could go either way, no strong preference. Interestingly, looks like there is quite a bit of discussion on the topic

https://github.com/microsoft/TypeScript-Handbook/issues/121

https://stackoverflow.com/questions/31876947/confused-about-the-interface-and-class-coding-guidelines-for-typescript/41967120#41967120

Reply:
Haha.  I guess there's no `I` in Team TypeScript.